.github/workflows/ci.yaml

```diff
@@ -59,6 +59,26 @@ jobs:
         run: |
           make unit-tests
 
+  int-test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+          cache-dependency-path: "**/*.sum"
+      - name: Install kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.24.0/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
```

Comment:
I've used the kind action for this sort of thing:
https://github.com/replicatedhq/pvmigrate/blob/main/.github/workflows/build-test.yaml#L37

```
      - name: Create k8s Kind Cluster
        uses: helm/kind-action@v1.10.0
```

Reply:
though I suppose that means you can't run it locally as easily

Reply:
my goal is that you can click run test in your IDE. id rather not make this depend on github actions